test/tests/unit/acq/test_daqtasks.py

```diff
@@ -67,6 +67,7 @@ def test_sync_finite(self):
                 tolerance = max(amp*0.1, 0.005) #noise floor
                 assert np.allclose(stim[10:],response[10:],rtol=0,atol=tolerance)
             
+    @unittest.skip("WHATEVER no hardware")
```

Comment:
We can leave the skips in while we are not using the chart function, but change the messages to reflect that we can skip them because the functionality is for future development